### ISSUE TITLE
Run command on startup?

### ISSUE THREAD

**Issue by defunkt** [alloy/kicker#2]
Is there a way to have kicker run its "execute" command on startup?

**Comment by alloy** [comment]
Are you using a .kick file or the -e option?

**Comment by defunkt** [comment]
Both? :P

I'm doing this in `.kick` right now:

```
Thread.new do
  sleep 3
  `touch test/haystack_test.rb`
end
```

I want to run my tests when Kicker starts. I'd do it manually but I want to use Kicker's growl goodies.

**Comment by alloy** [comment]
Added a startup chain which, as the name implies, is ran once Kicker has launched. Closed by 73fc6fe93dbb0ecee8cd693bcbfa4a482b753fdc

**Comment by defunkt** [comment]
Awesome, works great!
